test/controllers/auth_tokens_controller_test.rb

```diff
@@ -12,7 +12,15 @@ class AuthTokensControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
   end
 
-  test 'should create auth_token' do
+  test 'should allow user to authorize using the fallback header' do
```

Comment:
I only added this fallback test in one controller. I don't think we need to test this more than once.